frontend/src/index.ts

```diff
@@ -16,7 +16,7 @@ import { theme } from "./app-theme";
 import "./app-styles.sass";
 
 const isProduction = process.env.NODE_ENV === "production";
-const disableLogin = !!process.env.REACT_APP_DISABLE_LOGIN;
+const disableLogin = !!process.env.REACT_APP_DISABLE_LOGIN && JSON.parse(process.env.REACT_APP_DISABLE_LOGIN);
```

Comment:
```suggestion
const disableLogin = !!process.env.REACT_APP_DISABLE_LOGIN && JSON.parse(process.env.REACT_APP_DISABLE_LOGIN.toLowerCase());
```
https://stackoverflow.com/a/7833897 schlägt noch ein .toLowerCase vor.
Habs gerade aus probiert, sonst gibt eine Exception